### PR TITLE
add readmes to examples

### PR DESCRIPTION
**Pull request by drunkirishcoder**

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
@ari23 yes, we mostly use it for testing purposes, or like running an nf on a timer.

**Comment by zeeshanlakhani** [comment]
@ari23 also, super thankful for getting your review on this stuff!

File: examples/ping6d/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Ping6 daemon example
```
Comment (zeeshanlakhani) [inline examples/ping6d/README.md]: obviously, this now needs a minor update.

**Comment by ari23** [comment]
you're welcome!

**Comment by ari23** [comment]
found this very minor typo: 

```diff --git a/core/src/packets/icmp/v4/mod.rs b/core/src/packets/icmp/v4/mod.rs
index 100b324..70ada24 100644
--- a/core/src/packets/icmp/v4/mod.rs
+++ b/core/src/packets/icmp/v4/mod.rs
@@ -341,7 +341,7 @@ mod tests {
     }
 
     #[nb2::test]
-    fn parse_non_icmpv6_packet() {
+   fn parse_non_icmpv4_packet() {
         let packet = Mbuf::from_bytes(&UDP_PACKET).unwrap();
         let ethernet = packet.parse::<Ethernet>().unwrap();
         let ipv4 = ethernet.parse::<Ipv4>().unwrap();
```

File: examples/ping6d/README.md (hunk 1 above)
Comment (drunkirishcoder) [reply examples/ping6d/README.md]: rebased and updated.

**Pull request merged**